build/build-appveyor.cmd

```diff
@@ -2,7 +2,41 @@ ECHO APPVEYOR_REPO_BRANCH: %APPVEYOR_REPO_BRANCH%
 ECHO APPVEYOR_REPO_TAG: %APPVEYOR_REPO_TAG%
 ECHO APPVEYOR_BUILD_NUMBER : %APPVEYOR_BUILD_NUMBER%
 ECHO APPVEYOR_BUILD_VERSION : %APPVEYOR_BUILD_VERSION%
+ECHO CONFIGURATION : %CONFIGURATION%
 
 CALL build\tools\NuGet.exe restore src\Our.Umbraco.RobotsTxtEditor.sln
-CALL "%programfiles(x86)%\Microsoft Visual Studio\2017\Community\MSBuild\15.0\Bin\amd64\MsBuild.exe" build\package.proj
-REM CALL "%programfiles(x86)%\Microsoft Visual Studio\2017\Enterprise\MSBuild\15.0\Bin\amd64\MsBuild.exe" build\package.proj
+
+cd build
+
+SET toolsFolder=%CD%\tools\
+IF NOT EXIST "%toolsFolder%" (
+	MD tools
+)
+
+IF NOT EXIST "%toolsFolder%vswhere.exe" (
+	ECHO vswhere not found - fetching now
+	tools\NuGet.exe install vswhere -Version 2.6.7 -Source nuget.org -OutputDirectory tools
```

Comment:
@leekelleher this line looks to be doing the nuget install of vswhere (if hasn't already been downloaded)

All this extra code was copied from https://github.com/CrumpledDog/Umbraco-HealthCheckSlackNotificationMethod 

Jeavon recently changed that repo so people already on 2019 would be able to run the build task. And suggested we do the same. Who am I to argue? ;-)

Reply:
Doh! I totally missed that line. Gotcha, all pieced together (in my head) now. Thank you!

Reply:
Hehe well thank you for the questions as made me go back and really understand what code I'd added. Excellent mentoring there @leekelleher :)

Reply:
Just caught up with all this chat! Excellent work, LP! Also, thanks for putting all the steps up here for us to see. 👍